kvdb-memorydb/src/lib.rs

```diff
@@ -67,6 +67,21 @@ impl KeyValueDB for InMemory {
 						col.remove(&*key);
 					}
 				}
+				DBOp::DeletePrefix { col, prefix } => {
+					if let Some(col) = columns.get_mut(&col) {
+						use std::ops::Bound;
+						if prefix.is_empty() {
+							col.clear();
+						} else {
+							let start_range = Bound::Included(prefix.to_vec());
+							let end_range = Bound::Excluded(kvdb::end_prefix(&prefix[..]));
+							let keys: Vec<_> = col.range((start_range, end_range)).map(|(k, _)| k.clone()).collect();
+							for key in keys.into_iter() {
+								col.remove(&key[..]);
```

Comment:
this could be optimized by using `split_off` and `append`, but currently append is slow: https://github.com/rust-lang/rust/issues/34666 and I think memorydb is only used for tests and kvdb-web, so it's fine for now